Xyar changes batch 5

Xyarvius opened the pull request:
Hi @2kProjectEu4 @Glidar-dev,
i have another batch of mostly small changes (centering and aligning elements), this time mostly around unique religion and government mechanics.
Here is the full changelog: https://docs.google.com/document/d/1g3vGjARjzKosMFv1WKSP0l-643D-pq3yFXXCLizH3Y4/edit?tab=t.0#heading=h.5uft68pmn8sx
Apart from moving elements a few pixels, here are some bigger changes:
I added the titles to the edicts back:
![{1ECEF75F-AB71-431B-A733-B0BF1177B64A}](https://github.com/user-attachments/assets/0e80e562-a9f8-4804-b744-c0c9c50acf27)
I added a banner to the province history screen:
![{4A3BC91F-A0C5-406B-A551-D703D0250DE7}](https://github.com/user-attachments/assets/8a248b94-8b63-4add-96ae-bb1d1eab3ab9)
I changes the "War Score" sorting string in the peace deal screen to "Cost" so it fits and matches its tooltip:
![{BBA8EFC4-8F0C-44A5-9D16-578B53EF9F5E}](https://github.com/user-attachments/assets/d0c75347-6bf2-447a-9bbb-3f4cbe035183)
I fixed the naval missions background (which i messed up earlier i think)
![{CEDE0859-4D73-420B-93F1-404EA974C1AE}](https://github.com/user-attachments/assets/e1a69d73-ed2d-417b-84cf-895421c22f30)
I fixed the Natives building table and background:
![{EDADDF68-F412-45FC-A289-9E20C8010D96}](https://github.com/user-attachments/assets/18c75922-4fb6-49af-b650-c6b6ee22ab84)
I added an icon to Tribal Development in the Natives screen:
![{CDB435A4-F1AB-4BC1-912B-0BFA1F06F3A0}](https://github.com/user-attachments/assets/ff67a39d-d84b-40ee-a848-12c888a0f5b2)
I cleaned up some background sprites of the HRE screen, incresed resolution etc. and reduced the width of the "Reforms" box so it does not give the impression, that there could be more than 7 reforms:
![{4484701E-B9B5-4D06-91BE-6948604CA093}](https://github.com/user-attachments/assets/0a8880c3-d06a-408f-82d5-f5dd12200ac6)


Here is a patch for the tech+ideas submodule. It changes the Idea Select screen so that you can see the underlying window elements and it updates the idea pictures, since i found a way to add an upscaled version of them so they are not blurry anymore.
![{564AD751-44FD-4230-9A84-427F89E19A38}](https://github.com/user-attachments/assets/d21ce4cf-7295-4bc4-9d95-338b28e83825)
https://drive.google.com/file/d/1VBpc4iXa87zaPd_hsMegq3o9E4Jh4lZo/view?usp=sharing


Here is a patch for the upscaled minimap-submodule which just contains a small change from the main module:
https://drive.google.com/file/d/1AZ4rLtnQAfCj7Y7gROBJaaC97njezLuq/view?usp=sharing


Here is a patch for the decisions submodule. It contains the changes needed for the upscaled ideas pictures and a proposed change for the decision icons:
I'd like to change them from this
![{FBF158CD-0B77-4265-9CFC-E0027EB346D0}](https://github.com/user-attachments/assets/95c79a3d-7c89-428d-9a71-0bff59f80da8)
back to the original version
![{86FA5686-7225-4F64-9A87-AE1F37CED248}](https://github.com/user-attachments/assets/6d41b4b3-63f5-4cf8-8eaa-c19d8eb391bb)
because of consistency regarding the icons in other places (e.g. mission tree) and artstyle.
If you don't like it, you can just delete the respective images from the patch files (the x and y values stayed the same):
https://drive.google.com/file/d/1FK-onSWrtDAbTc95tnrtfBIHMg8HMxI_/view?usp=sharing


Then there is a patch for the regional & urban terrain pictures submodule. It just contains one additional upscaled picture which was added in the last months:
https://drive.google.com/file/d/1xjJ5nCSZKrlMeasSoTZDPXvK0RvXCtob/view?usp=sharing

And lastly, in the branch mission_banner_color_adjustment i have a proposal to change the mission banner colors slightly so that they are more in line with the general color palette of all other banners in the game. If you dont like it, you can just ignore it, no big deal.
![image](https://github.com/user-attachments/assets/9be37576-fac0-49fe-8285-04635310e9ce)


https://github.com/2kProjectEu4/Proper-2K-UI-Enhanced/tree/mission_banner_color_adjustment

Commit:
- made adjustments to HRE screen

Commit:
Merge branch 'main' into xyar_changes_batch_5

Commit:
- minor adjustments to country modifier list, hre screen and ideas screen

Commit:
- minor adjustments to hints

Commit:
- made small adjustments to tribal UI elements

Commit:
- slight changes to empire of china, parliament, faction screens

Commit:
- fixing several small things

Commit:
upscaled ideas pictures for idea submod

Commit:
- small bugfixes

Commit:
- updates flagship designer
- small adjustments here and there

Commit:
- fix artifacts in extended hre screen

Commit:
- updated some releigion screens
- updated nation designer

2kProjectEu4 commented:
Hello @Xyarvius. I like how it looks. I will take a look soon.

2kProjectEu4 commented:
Hello @Xyarvius,
I looked at the changes and fixed some small issues:
- Changed parliament effect max height; The missionary effect was not fully displayed
![Captură de ecran 2025-05-10 213853](https://github.com/user-attachments/assets/f78513d6-6615-4a4d-810d-702727c133e1)

- Fixed hre members list right cliping
![Captură de ecran 2025-05-11 142242](https://github.com/user-attachments/assets/e17a48e7-7262-4bf2-aa33-7b3d9689acbe)
- Reverted change of hre header ( its not movable if we replace guiButtonType with iconType)

- Regarding the imperial authority values, I feel like the position is not ideal when its maxed out. 
![Captură de ecran 2025-05-11 144750](https://github.com/user-attachments/assets/155be181-684f-41a4-b080-0683764c4aa1)

- Changed scale value of the idea icon size for base mod.
Previous:
![small images](https://github.com/user-attachments/assets/6972d47c-260b-4df0-8789-5bbf99fd9fcf)
Now:
![Captură de ecran 2025-05-11 145406](https://github.com/user-attachments/assets/6403b754-7961-4bb2-ac27-d1cf37a705aa)

One thing i reverted was the changes for Pause because every time I paused, I found myself searching for the pause status and I was feeling unsure whether it had actually paused or not. I could not get used to the new placement.

Xyarvius commented:
@2kProjectEu4 Thanks for looking over it and fixing the stuff i missed or broke. And i actually agree with the revert of the pause banner. I was unsure about that myself.

Here is a branch with a handfull of other fixes:
- moved the puase banner a few pixels to the center
- added tha idea background from the tech+idea submod matching the now bigger idea picture
- changed the max size of some strings as they either clipped or had more place
- made provinceview bg sprite consistent for own and foreign countries
- reduced height of speedcontrols hitbox so it does not overlap with the points (so the tooltip can be shown properly)

Branch: https://github.com/2kProjectEu4/Proper-2K-UI-Enhanced/tree/small_fixes
